cpp/example_code/neptune/describe_db_clusters.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+    http://aws.amazon.com/apache2.0/
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+
+#include <aws/core/Aws.h>
+#include <aws/neptune/NeptuneClient.h>
+#include <aws/neptune/model/DescribeDBClustersRequest.h>
+#include <aws/neptune/model/DescribeDBClustersResult.h>
+#include <iostream>
+
+/**
+ * Describes Neptune db cluster based on command line input
+ */
+
+int main(int argc, char **argv)
+{
+  if (argc != 1)
+  {
+    std::cout << "Usage: describe_db_cluster";
+    return 1;
+  }
+  Aws::SDKOptions options;
+  Aws::InitAPI(options);
+  {
+    Aws::Neptune::NeptuneClient neptune;
+
+    Aws::Neptune::Model::DescribeDBClustersRequest ddbc_req;
+
+    auto ddbc_out = neptune.DescribeDBClusters(ddbc_req);
+
+    if (ddbc_out.IsSuccess())
+    {
+      std::cout << "Successfully described db clusters request:";
+
+      for (auto val: ddbc_out.GetResult().GetDBClusters())
+      {
+        std::cout << " " << val << std::endl;
```

Comment:
This doesn't build for me.  I changed it to:

std::cout << " " << val.GetDBClusterIdentifier() << std::endl;

Reply:
I have updated the script, please let me know if you face issues in running the scripts.